### DISCUSSION TITLE
Appdata/Roaming/Code/Service Worker/CacheStorage folder size is multiple GBs

### DISCUSSION THREAD

**Discussion by TryerGit** [microsoft/vscode-cpptools#10409] [Q&A]
Hello,

I use VSCode mostly only for its C++ extension. The folder `C:\Users\TryerGit\AppData\Roaming\Code\Service Worker\CacheStorage` is more than 10 GB in size. Can it be clarified if the contents of this folder are used by the C++ extension and whether these can be safely deleted? Is there a way to set it up so that it does not grow so big?

Image attached below:
![vscode](https://user-images.githubusercontent.com/70434658/213747003-b531f5a9-1d94-4862-a44b-3cd4bc2ca14a.jpg)

Thank you.

**Comment by sean-mcmanus** [comment]
@TryerGit I'm not sure. Can you expand the CacheStorage folder and look for any ipch or .db files from our extension?

**Reply by sean-mcmanus** [reply]
From what I can tell, our extension doesn't drop anything big to that folder by default, i.e. you should look into what files are using the storage.

**Comment by Junyi42** [comment]
> Hello,
> 
> I use VSCode mostly only for its C++ extension. The folder `C:\Users\TryerGit\AppData\Roaming\Code\Service Worker\CacheStorage` is more than 10 GB in size. Can it be clarified if the contents of this folder are used by the C++ extension and whether these can be safely deleted? Is there a way to set it up so that it does not grow so big?
> 
> Image attached below: ![vscode](https://user-images.githubusercontent.com/70434658/213747003-b531f5a9-1d94-4862-a44b-3cd4bc2ca14a.jpg)
> 
> Thank you.

Same problem here. Deleted anything 3 months ago. So far so good.

btw, what's the program are you using to show the proportion of different folder, it looks great :-)

**Reply by TryerGit** [reply]
@Junyi42 

That is WinDirStat.

**Reply by Junyi42** [reply]
Thanks!
